<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgModbusFunction</class>
 <widget class="QDialog" name="DlgModbusFunction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Modbus Functions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_newRecord">
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="grp_new">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>Yeni Tip</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Tip</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Fonksiyon</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtAddType">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>45</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtAddFunction">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>45</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAdd">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCancel_2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>47</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grp_edit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>DÃ¼zenle</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Tip</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Fonksiyon</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtSaveType">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>45</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="edtSaveFunction">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>45</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCancel">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>45</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSave">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
